rules/pa/pa-phonetic.js

```diff
@@ -5,11 +5,11 @@
 		id: 'pa-phonetic',
 		name: 'Punjabi Phonetic',
 		description: 'Punjabi Phonetic',
-		date: '2012-10-16',
+		date: '2022-08-08',
 		URL: 'http://fedoraproject.org/wiki/I18N/Indic/PunjabiKeyboardLayouts',
```

Comment:
Can you update this to: https://fedoraproject.org/wiki/I18N/Indic/PunjabiKeyboardLayouts